README.md

```diff
@@ -50,6 +50,8 @@ a web browser, or to serve the book locally, type:
 $ mdbook serve
 ```
 
+The default address to view the book will be located at [http://localhost:3000](http://localhost:3000) .
```

Comment:
might have to specify `http://localhost:3000` vs a link so it doesn't break the CI

Reply:
@wheresaddie I think this is fine. Because if they run the book locally, they can now just click this link to access it.

Instead, I should create a ticket to have the link-checker ignore links that begin with `localhost`.

Reply:
yes, great suggestion @jimmychu0807 please proceed with the CI ticket to ignore localhost